src/lib/common/StreetViewController.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  import type { MapMouseEvent } from "maplibre-gl";
+  import cameraCursorUrl from "../../../assets/camera_cursor.svg?url";
+  import { map, userSettings } from "../../stores";
+  import { Radio } from "../govuk";
+  import SecondaryButton from "../govuk/SecondaryButton.svelte";
+  import { getRoadLayerHelpers, type LayerHelper } from "../maplibre";
+  import CollapsibleCard from "./CollapsibleCard.svelte";
+  import { ExternalLink } from "./";
+
+  export let displayEnableButton: boolean = false;
```

Comment:
Small code style nits, we could omit `: boolean` when there's a default value and the type is really clear. And generally we've been doing `T[]` instead of `Array<T>`, but I'm not really opinionated about it. Let's leave it for now, and maybe setup eslint or similar in the future, configure it how we like, and rely on it for enforcement